sycl/test/check_device_code/ap_fixed.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -I %sycl_include -S -emit-llvm -fsycl-device-only %s -o - -Xclang -disable-llvm-passes -Xclang -no-enable-noundef-analysis | FileCheck %s
+// RUN: %clangxx -I %sycl_include -S -emit-llvm -fno-sycl-early-optimizations -fsycl-device-only %s -o - -Xclang -disable-llvm-passes -Xclang -no-enable-noundef-analysis | FileCheck %s
```

Comment:
Why do we need `-fno-sycl-early-optimizations`? I'd think we mostly care about what the code is after default optimizations pipeline...